config.py

```diff
@@ -0,0 +1,134 @@
+# flake8: noqa
+import os
+
+from ras_party.cloud.cloudfoundry import ONSCloudFoundry
+
+
+cf = ONSCloudFoundry()
+
+
+class Config(object):
+
+    NAME = os.getenv('RAS-PARTY', 'ras-party')
+    VERSION = os.getenv('VERSION', '1.0.0')
+    SCHEME = os.getenv('http')
+    HOST = os.getenv('HOST', '0.0.0.0')
+    PORT = os.getenv('PORT', 8081)
+    DEBUG = os.getenv('DEBUG', False)
+    LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
+    SECRET_KEY = os.getenv('SECRET_KEY', 'aardvark')
+    EMAIL_TOKEN_SALT = os.getenv('EMAIL_TOKEN_SALT', 'aardvark')
+    PARTY_SCHEMA = os.getenv('PARTY_SCHEMA', 'ras_party/schemas/party_schema.json')
+
+    if cf.detected:
+        DATABASE_SCHEMA = 'partysvc'
+        DATABASE_URI = cf.db.credentials['uri']
+    else:
+        DATABASE_SCHEMA = os.getenv('DATABASE_SCHEMA', 'partysvc')
```

Comment:
Does the schema need to be configurable?

Reply:
I'm not sure myself - probably not with our current deployment.